### DISCUSSION TITLE
Is there a mosaic endpoint that you can use to extract values at a point?

### DISCUSSION THREAD

**Discussion by hrodmn** [stac-utils/titiler-pgstac#144] [Q&A]
I would like to add the ability to query values at a point in a web map. Is there an existing endpoint for querying values at point location from a `mosaic`? I did a quick scan of the docs today and don't see an obvious candidate. The `/statistics` endpoint might be able to do it but I think this is a simpler operation than what happens in `/statstics`!

**Comment by vincentsarago** [comment]
There is not! but it should be fairly easy to add

There is already the code in the backend: https://github.com/stac-utils/titiler-pgstac/blob/e06ec4271fc00b035b184bb34a2f258cf4111943/titiler/pgstac/mosaic.py#L328-L367

we can follow https://github.com/developmentseed/titiler/blob/main/src/titiler/mosaic/titiler/mosaic/factory.py#L631-L678 for the endpoint
